### ISSUE TITLE
TableBatchActions: items selected string and cancel button i18n issues

### ISSUE THREAD

**Issue by nicoxxie** [carbon-design-system/carbon-components-react#780]
## Detailed description

for screen cap see https://github.ibm.com/IAM/access-management/issues/4919#issuecomment-5281214

> Describe in detail the issue you're having.
Currently the number selected text looks like this.
```
 <p className="bx--batch-summary__para">
          <span>{totalSelected}</span>{' '}
          {totalSelected > 1
            ? t('carbon.table.batch.items.selected')
            : t('carbon.table.batch.item.selected')}
        </p>
```
this is not the right behavior for many languages since the number may be in the middle or after the text.

Also, cancel button is not globalized.
```
<button className="bx--batch-summary__cancel" onClick={onCancel}>
          Cancel
        </button>
```

> Is this a feature request (new component, new icon), a bug, or a general issue?
bug
> Is this issue related to a specific component?
TableBatchActions

> What did you expect to happen? What happened instead? What would you like to see changed?
globalized correct

> What browser are you working in?
all

> What version of the Carbon Design System are you using?
5.29.4

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?
IAM cloud
## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

* Screenshots or code
* Notes

**Comment by joshblack** [comment]
Hi @nicoxxie! 👋 Thanks for bringing up these i18n concerns. Definitely need to add support for this into DataTable. Do you have some details you could provide on what tools/libraries you all are using for i18n so we can help identify intermediate workarounds while we improve in this area?

**Comment by nicoxxie** [comment]
Hi @joshblack, we are using react-i18next and i18next.
For the texts maybe we can do the same like what pagination component is doing? https://github.com/carbon-design-system/carbon-components-react/blob/master/src/components/PaginationV2/PaginationV2.js#L36
